content/sql/concepts/operators/terms/null-safe-equal/null-safe-equal.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'NULL SAFE EQUAL'
+Description: 'Performs an inequality comparison similar to the '=' operator, using the logic of an XNOR gate. Can accept NULL as a value and will not output NULL.'
+Subjects: 
+  - 'Computer Science'
+Tags: 
+  - 'Operators'
+  - 'Database'
+  - 'Queries'
+  - 'Tables'
+CatalogContent: 
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The **'NULL SAFE EQUAL'** operator performs an equality comparison like the '=' operator. However it can also accept NULL values and never returns NULL, instead always returning TRUE or FALSE. It uses the logic of an XNOR gate to determine the value returned.
```

Comment:
```suggestion
The **`NULL SAFE EQUAL`** operator performs an equality comparison like the '=' operator. However, it can also accept NULL values and never returns NULL, instead always returning TRUE or FALSE. It uses the logic of an XNOR gate to determine the value returned.
```